StreamChat.xcodeproj/project.pbxproj

```diff
@@ -8456,6 +8456,7 @@
 		};
 		A3B78F1B282AB07100348AD1 /* SwiftLint */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
Needed?

Reply:
Yeah, to remove a SwiftLint warning, same as we've done for SDKs and DemoApp.